bats/tests/helpers/os.bash

```diff
@@ -84,16 +84,29 @@ sudo_needs_password() {
 
 supports_vz_emulation() {
     if is_macos; then
+        if [[ -n ${_RD_SUPPORTS_VZ_EMULATION:-} ]] || load_var _RD_SUPPORTS_VZ_EMULATION; then
+            if is_true _RD_SUPPORTS_VZ_EMULATION; then
```

Comment:
`is_true` does not evaluate its argument in any form; it just does a regex check against the string directly.

```suggestion
            if is_true "${_RD_SUPPORTS_VZ_EMULATION:-}"; then
```

Reply:
Good catch; I didn't notice because this always evaluated to true (because `_RD_…` doesn't look false).